### DISCUSSION TITLE
website dark theme

### DISCUSSION THREAD

**Discussion by usman26392** [vercel/next.js#73385] [Help]
### Summary

I just want to ask about website's dark theme , what is in VS code?

![image](https://github.com/user-attachments/assets/65ae2dcb-8e98-4b65-8a97-c3508e6c0437)

**Comment by icyJoseph** [comment]
I think the closest is something like this, https://marketplace.visualstudio.com/items?itemName=guilhermerodz.veist

**Reply by usman26392** [reply]
I need an exact one.

**Reply by icyJoseph** [reply]
If I remember correctly, their theme is "closed source", or rather not available, unless you map it out yourself.

You can also use this as a reference I guess: https://geist-ui.dev/en-us/components/code

**Reply by usman26392** [reply]
I have found very close to vercel theme.
https://marketplace.visualstudio.com/items?itemName=gantoreno.vscode-vercel-theme
